ci/download_buildozer.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
```

Comment:
Note: this file is mainly copy/paste from `ci/download_buildifier.sh`.  I don't know shell scripts well enough, but the only difference is replacing s/buildifier/buildozer